include/oc_trace_pb.hrl

```diff
@@ -0,0 +1,221 @@
+%% -*- coding: utf-8 -*-
```

Comment:
should automatically generated files be committed? Not super objected to it just curious for the reasons.